ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -1306,6 +1306,14 @@ class JITModBuilder {
               continue;
             }
 
+            // Vararg related intrinsics appear to be always inlined.
```

Comment:
This comment doesn't quite make sense as-is: I *think* it wants to say "Intrinsics with var args are always inlined so ..." but I don't know what the "..." should be!

Reply:
Intrinsics are a bit of a tricky thing since they may or may not be inlined. We thus have this `yk.intrinsic.inlined` metadata that we attach to the AOT IR to know which one it is. `memcpy` is an example for such an intrinsic. But then there are others like `va_start` and `mul_with_overflow`, which as far as I can tell never become a call and are always turned into inlined machine code. I made the decision to handle them in yk rather than using the `yk.intrinsic.inlined` thing, since it gives us more flexibility for adding similar intrinsics and avoids having to change ykllvm when it's not strictly necessary.